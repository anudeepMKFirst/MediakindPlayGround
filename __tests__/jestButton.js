import React from 'react';
import Button from "./../button.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Button name="temp"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});