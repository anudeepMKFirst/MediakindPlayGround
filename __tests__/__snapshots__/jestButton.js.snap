// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<YiPushButtonView
  name="temp"
  onCompositionDidLoad={[Function]}
  onPress={[Function]}
  onTimelinesLoaded={[Function]}
>
  <TimelineRef
    name="FocusIn"
    onAborted={[Function]}
    onCompleted={[Function]}
    onCompletedForward={[Function]}
    onCompletedReverse={[Function]}
    onCompositionDidLoad={[Function]}
    onPaused={[Function]}
    onPausedForward={[Function]}
    onPausedReverse={[Function]}
    onPlay={[Function]}
    onPlayForward={[Function]}
    onPlayReverse={[Function]}
    onStarted={[Function]}
    onStartedForward={[Function]}
    onStartedReverse={[Function]}
  />
</YiPushButtonView>
`;
